### DISCUSSION TITLE
Regarding the supported languages, is it possible to open the public to assist in translation?

### DISCUSSION THREAD

**Discussion by SnowViktor** [community/community#44380] [Mobile] [answered]
# GitHub Mobile's language:
_Chinese is only Simplified Chinese._ Adding **traditional Chinese** only needs to translate the text, and the grammar does not need to be changed too much.

**Comment by stevepopovich** [comment, marked as answer]
Hi @Code-Xuebao thanks for the feedback. I have passed this on to the team for consideration.
